nevergrad/benchmark/experiments.py

```diff
@@ -257,9 +257,11 @@ def mldaas(seed: Optional[int] = None) -> Iterator[Experiment]:
                             yield xp
 
 
+# def arcoating(seed: Optional[int] = None) -> Iterator[Experiment]:
 @registry.register
-def arcoating(seed: Optional[int] = None) -> Iterator[Experiment]:
+def arcoating() -> Iterator[Experiment]:
     func = ARCoating()
+    seed = 12
```

Comment:
Why do you need this? Either this isn't used, or you're introducing a seemingly unrelated functional change.

Reply:
This was just a test that I removed probably a second after your comment